src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_ListViewBase.cs

```diff
@@ -3806,6 +3806,43 @@ public async Task When_SelectionMode_Is_Multiple()
 			Assert.AreEqual(vsg.CurrentState?.Name, "MultiSelectEnabled");
 		}
 
+#if HAS_UNO
+		[TestMethod]
+		public async Task Valid_MultipleSelectionMode_ValidSelectionStates()
+		{
+			const string msc = "MultiSelectCheck";
+			var lvi0 = new ListViewItem
+			{
+				Content = "child 1"
+			};
+			var SUT = new ListView
+			{
+				SelectionMode = ListViewSelectionMode.Multiple
+			};
+			SUT.Items.Add(lvi0);
+
+			WindowHelper.WindowContent = SUT;
+			await WindowHelper.WaitForLoaded(SUT);
```

Comment:
Can you try adding `await WindowHelper.WaitForIdle();` after this `WaitForLoaded` call? It might be a lifecycle-related issue on iOS where this extra call solves it.

Reply:
Its created new issues

Reply:
May I revert the last commit?

Reply:
I think it didn't. It seems it actually fixed the issue. The current failure appears to just be a timeout that would be resolved when the job restarts. I'll restart it once the iOS tests stage completes and see.

Reply:
> May I revert the last commit?

Please don't. Let's wait for CI to complete first as I think the last commit fixed the test failure.